README.md

```diff
@@ -1,6 +1,24 @@
-# OpenEuropa multivalue_form_element
+# Multi-value form element
+
+Provides a multi-value form element that wraps other single or sets of form elements.
```

Comment:
Provides a multi-value form element that wraps single form elements or sets of form elements.